junit-platform-engine/src/main/java/org/junit/platform/engine/discovery/NestedMethodSelector.java

```diff
@@ -154,4 +155,13 @@ public int hashCode() {
 		return Objects.hash(nestedClassSelector, methodSelector);
 	}
 
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this) //
+                .append("enclosingClassNames", getEnclosingClassNames()) //
+                .append("nestedClassName", getNestedClassName()) //
+                .append("methodName", getMethodName()) //
+                .toString();
```

Comment:
Please add `methodParameterTypes`.

Reply:
Should I also add `getMethodParameterTypes()` then? I was wondering about that

Reply:
I guess so. 👍